server/channels/app/platform/web_conn.go

```diff
@@ -393,8 +388,6 @@ func (wc *WebConn) GetSession() *model.Session {
 
 // SetSession sets the session of the connection.
 func (wc *WebConn) SetSession(v *model.Session) {
-	// Clone the session first as WebConn takes ownership of the object
-	// and the web.Hub will return it to the [sync.Pool] once the WebConn gets removed.
 	if v != nil {
 		v = v.DeepCopy()
```

Comment:
Do we need to copy the session here?

Reply:
Yes, this is unrelated to the sync.Pool implementation: https://github.com/mattermost/mattermost/pull/16103. So it should still be there.